pkgs/top-level/perl-packages.nix

```diff
@@ -20404,7 +20404,8 @@ let
       url = mirror://cpan/authors/id/T/TO/TODDR/XML-Parser-2.46.tar.gz;
```

Comment:
oops
The PR was against 2.46
But it is now `2.44` in `staging`
So it should be upgraded back to `2.46`

Reply:
https://github.com/NixOS/nixpkgs/pull/73954/commits/ca04f87ff5c139452fc0981fad861c1f0b6db99d is to be reverted